### ISSUE TITLE
Stage deploy failing

### ISSUE THREAD

**Issue by hoosteeno** [mdn/viewsourceconf#187]
```
<snip>
Removing intermediate container bb1b0c61bb32
Successfully built c33b226eb805
docker tag -f quay.io/mozmar/viewsourceconf\:5940119 127.0.0.1:5001/viewsourceconf-stage\:5940119
unknown shorthand flag: 'f' in -f
See 'docker tag --help'.
make: *** [push-private-registry] Error 1
</snip>
```

https://ci.us-west.moz.works/view/viewsourceconf/job/viewsourceconf_deploy_stage/274/console

**Comment by hoosteeno** [comment]
@jgmize could you help us with this?

**Comment by jgmize** [comment]
Fixed by 9d33474 deployed in https://ci.us-west.moz.works/view/viewsourceconf/job/viewsourceconf_deploy_stage/275/console
